packages.config

```diff
@@ -9,6 +9,7 @@
   <package id="EPiServer.Commerce.Core" version="13.0.0" targetFramework="net461" />
   <package id="EPiServer.Framework" version="11.11.1" targetFramework="net461" />
   <package id="EPiServer.Framework.AspNet" version="11.11.1" targetFramework="net461" />
+  <package id="HtmlAgilityPack" version="1.11.24" targetFramework="net461" />
```

Comment:
You also need to add this to the nuspec file. Round down the version number to the lowest minor build version.

Reply:
Thanks, I added it with 1.11.0 version to unspecified.